src/client/rest/RESTMethods.js

```diff
@@ -914,8 +914,10 @@ class RESTMethods {
       .then(() => message);
   }
 
-  getMessageReactionUsers(message, emoji, limit = 100) {
-    return this.rest.makeRequest('get', Endpoints.Message(message).Reaction(emoji, limit), true);
+  getMessageReactionUsers(message, emoji, options) {
+    const queryString = (querystring.stringify(options).match(/[^=&?]+=[^=&?]+/g) || []).join('&');
```

Comment:
`querystring.stringify(options)` is enough, the extra stuff is redundant.

Reply:
This regex is dropping emtpy query string paramters `after=&before=&limit=100
` -> `limit=100`.
Discord is rejects those: `after: Value "" is not snowflake.`

Reply:
Oh, my bad.